src/bin/find.js

```diff
@@ -8,6 +8,7 @@ var options = {
   getAllAvailableRules: ['all-available', 'a'],
   getUnusedRules: ['unused', 'u'],
   n: ['no-error'],
+  noCore: ['nocore', 'C'],
```

Comment:
Oh yes, forgot to mention that. I tried `no-core`, but could not get it to work. I thought it was because of the dash `-`, but now that I look closer, `all-available` works.

I think I just figured it out:
`yargs` tries to be smart, and when it sees something like `no-xyz`, then it sets `xyz` to false

See the state of `argv` when I use `no-core`
```bash
./bin/find.js --no-core      
{ _: [],
  getCurrentRules: false,
  current: false,
  c: false,
  getPluginRules: false,
  plugin: false,
  p: false,
  getAllAvailableRules: false,
  'all-available': false,
  a: false,
  allAvailable: false,
  getUnusedRules: false,
  unused: false,
  u: false,
  n: false,
  'no-error': false,
  noError: false,
  nc: false,
  'no-core': false,
  C: false,
  noCore: false,
  verbose: false,
  v: false,
  core: false,
  '$0': '../../dev/eslint-find-rules/src/bin/find.js' }
```

noCore and friends are false, though they should be set to true, but `core` appeared and is set to `false`.
I reproduce the same thing with `no-error`, which actually does not work as expected: the bin fails when called with the long form `--no-error` but works fine with `-n`.

I'll try to find a nice solution to this without changing the `no-error` param, and doing `nocore` --> `no-core`